engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/EpbContext.java

```diff
@@ -63,7 +63,7 @@ private static void initializeLanguages(
       return;
     }
     var log = environment.getLogger(EpbContext.class);
-    log.log(Level.FINE, "Initializing languages {0}", langs);
+    log.log(Level.FINE, "Initializing languages {}", langs);
```

Comment:
Are you sure `{}` works? It is certainly not standard when using `LogRecord` standard formatter. As far as I know it didn't work before - as such I had to use `{0}`.

Reply:
It _worked_ in a sense that all `LogRecord`s were forwarded to logback unformatted and logback recognized those. But when using plain jul this didn't work indeed (which was visible in tests). I reverted now and doing the `s/{\d+}/{}` before forwarding to logback.